build/commands/lib/config.js

```diff
@@ -1137,6 +1138,10 @@ Object.defineProperty(Config.prototype, 'defaultOptions', {
       env.VSCMD_SKIP_SENDTELEMETRY = '1'
     }
 
+    if (this.isCI && this.skip_download_rust_toolchain_aux) {
+      env.SKIP_DOWNLOAD_RUST_TOOLCHAIN_AUX = '1'
```

Comment:
why do we want to skip this?

Reply:
To prevent the CI job that builds these packages from attempting to also download them during sync.